ironfish-cli/src/commands/miners/start.ts

```diff
@@ -33,6 +37,11 @@ export class Miner extends IronfishCommand {
       char: 'a',
       description: 'the public address to receive pool payouts',
     }),
+    richOutput: Flags.boolean({
```

Comment:
Minor but could the flags be like `--verbose` vs. `--quiet`? `--no-richOutput` seems very unintuitive (to me).

Reply:
`--verbose` exists and is used for, well, more verbose output. This is a fancy thing that hangs at the bottom of the terminal, which does need to able to be turned off because it doesn't play nice with certain TTY emulators, etc. I also don't love the name, so open to suggestions

Reply:
Yeah, not sure then. `richOutput` makes me think of Ron Burgundy and "Corinthian leather" ^^